src/filesystem/impls/filer/lib/FileImport.js

```diff
@@ -52,10 +53,14 @@ define(function (require, exports, module) {
 
         var strategy = chooseImportStrategy(source);
 
+        PreferencesManager.set("livePreviewAutoReload", false);
```

Comment:
Add a comment above this indicating that we want to pause live preview updates until all files have been imported, due to import order vs dependency order and spurious 404s.